src/App.css

```diff
@@ -27,6 +27,7 @@ body {
   transform-origin: 50% 100%;
   transform: perspective(3000px) rotateX(32deg);
   width:100%;
+  pointer-events:none;
```

Comment:
I see, so the svg was becoming the target of a mouse event, and obscuring the actual button underneath, and so this stops that?

Reply:
Yes, I believe it was masking the button.